sources/Outputs/include/Dyd.h

```diff
@@ -121,12 +124,24 @@ class Dyd {
   /**
    * @brief Create black box model for generator
    *
+   * @param collection the collection for which the connections will be added
```

Comment:
Do you have any idea ? I can't find one

Reply:
dynmicModelsToConnect?

Reply:
Ah, I thought you were talking about the description of the parameter collection. The name "collection" is already used in the function write of the Dyd, so it would imply changing it all over the file.

Reply:
My opinion is to do it because collections is generic and makes it harder to understand an algorithm that is starting to be complex